### ISSUE TITLE
Search Component design feedback from USA.gov

### ISSUE THREAD

**Issue by mariamarrero** [uswds/uswds#1640]
Issue more for consideration. Also, wondering if other users have provided feedback on this.

In USA.gov, we customized the WDS search box to reflect:

- a wider input field to give desktop users enough space for their query and make it easier for them to review or edit it. 
- visible expanded search box: now that we have a less cluttered header, we kept a visible input field (as opposed to just having the icon). Users tend to find it faster and easier. 

Finally, we are starting to conduct low fidelity UT on new header/footer. So far, we hear that users are having a hard time finding the search in mobile and some don't expect to find it under "Menu". We will continue to test this with a more functional prototype and report back. Just sharing some early feedback.

You might have seen some of these resources, but figured it would not hurt to include them here. Or they could perhaps be listed under the "For further reading" section: 
- https://www.nngroup.com/articles/search-visible-and-simple/
- https://www.nngroup.com/articles/magnifying-glass-icon/
- https://www.smashingmagazine.com/2009/09/10-useful-usability-findings-and-guidelines/#6-the-ideal-search-box-is-27-characters-wide)

**Comment by shawnbot** [comment]
This is related to #1435 and #1596.

**Comment by maya** [comment]
I couldn't find any info in the header research docs recommending the search be closed: 
🔒 https://drive.google.com/drive/u/0/folders/1hsP34v5eAv7t0N1_0HYIj2LRl2lGPoxF

Older header designs have is open by default:
### Simple
![headers_navigation_ui_v2_simple_1024](https://user-images.githubusercontent.com/5249443/42969071-e1c8a5e0-8b59-11e8-9eb4-8d8c107d7dda.png)

### Complex
![headers_navigation_ui_v2_complex_1024](https://user-images.githubusercontent.com/5249443/42969126-042fa7dc-8b5a-11e8-8534-a1dc5016dc13.png)

**Comment by maya** [comment]
I think this feedback is right on, @mariamarrero. I'm unable to find anywhere information that suggests hiding the search field is a good pattern and https://www.nngroup.com/articles/magnifying-glass-icon/ states:
 
>Above all, retain an open text-entry field next to the icon in the desktop version of a website.

The navigation in the image above is the layout I'd recommend we follow. 

Next steps: determine if implementing this would be a breaking change (2.0) or minor change.

Thanks again, @mariamarrero!

**Comment by maya** [comment]
This was fixed in #2606 and will be available in the 2.0 release 🎉

**Comment by mariamarrero** [comment]
Great! thanks @maya for the update!
